examples/knative/python/kn-py-vm-attr/handler.py

```diff
@@ -4,85 +4,40 @@
 from vcenter import Session
 from datetime import date
 
-logging.basicConfig(level=logging.DEBUG,format='%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s')
+logging.basicConfig(
+    level=logging.DEBUG,
+    format='%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s'
+)
 
 app = Flask(__name__)
-@app.route('/', methods=['GET','POST'])
+@app.route('/', methods=['POST'])
 def echo():
```

Comment:
better rename this

Reply:
I think I could have been running this with `echo` for years before discovering it !